src/configs/formatters/getTsConfig.ts

```diff
@@ -0,0 +1,58 @@
+import StyleDictionary, { Config, filter, formatHelpers } from 'style-dictionary'
+import { DesignToken } from 'style-dictionary/types/DesignToken'
+
+export function getWebTsConfig(sd: typeof StyleDictionary, brand: string, theme: string): Config {
+  const destination = `index.${theme}.web.ts`
+  const sizeTransform = 'size/pxToRem'
+
+  return getTsConfig(sd, theme, brand, sizeTransform, destination)
+}
+
+export function getMobileTsConfig(
+  sd: typeof StyleDictionary,
+  brand: string,
+  theme: string
+): Config {
```

Comment:
si on extrait le type de la fonction comme dit dans un commentaire précédent, on peut simplifier le typage

```suggestion
export const getMobileTsConfig: Formatter = (sd, brand, theme) {
```

Reply:
Ah oui, super ça! J'ai changé les noms des types parce que "Formatter" est déjà le type de style-dictionary.